### ISSUE TITLE
BobWin.exe crashes on wiki creation

### ISSUE THREAD

**Issue by amreus** [OokTech/TW5-Bob#155]
Using BobWin.exe 1.6.2 on Windows 10.

Starting from a clean directory, run bobWin.exe

Go to ControlPanel -> Bob -> Wiki Management -> Create Wiki

Select from an edition, select empty edition, enter a name of "a" (no quotes)

Here is the text from the command line:

```
C:\Users\amreus\Downloads>bob
NodeSettings - Parsed raw settings.
TiddlyWiki version 5.1.22 with Bob version 1.6.2
Bob saver server running on 127.0.0.1:61192
Serving on 127.0.0.1:8080
(press ctrl-C to exit)
new connection
data {
  type: 'createNewWiki',
  wiki: 'RootWiki',
  path: 'a',
  wikiName: 'a',
  edition: 'empty',
  id: 'b2',
  source_connection: 0,
  decoded: true
}
Copied edition 'empty' to C:\Users\Jim\Downloads\Wikis\a

Added route /^\/a\/?$/
Wrote settings file
$:/plugins/OokTech/Bob/NodeWikiCreationHandlers.js:475
            alert: 'Created wiki ' + name,
                                     ^

ReferenceError: name is not defined
    at Timeout.callback [as _onTimeout] ($:/plugins/OokTech/Bob/NodeWikiCreationHandlers.js:475:38)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)

C:\Users\amreus\Downloads>
```

**Comment by inmysocks** [comment]
This will be fixed with version 1.6.2r2 which is currently uploading.

**Comment by amreus** [comment]
All good, thanks.
